assets/js/arrayProcessingTool.js

```diff
@@ -0,0 +1,94 @@
+const inputArrayHTMLElement=document.getElementById('array');
+const resultHTMLElement=document.getElementById('result');
+const searchButtonHTMLElement=document.getElementById('search-button');
+const sequenceButtonHTMLElement=document.getElementById('sequence-button');
+const maximalSumButtonHTMLElement=document.getElementById('maximal-sum-button');
+
+inputArrayHTMLElement.onchange=()=>{
+  object.arr=inputArrayHTMLElement.value.split(' ').map(el=>Number(el));
+  console.log(object.arr);
+  if(object.arr.includes(NaN)){
+      resultHTMLElement.style.backgroundColor='#f85454'
+      resultHTMLElement.textContent='some element(s) in array is not correct. Result could fail';
+  }
+};
+
+const object={
```

Comment:
Give an understandable name for this variable

Reply:
fixed

Reply:
fixed